netlink-packet-route/src/rtnl/link/nlas/link_infos.rs

```diff
@@ -1345,6 +1355,54 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoIpVlan {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum InfoVrf {
+    Unspec(Vec<u8>),
```

Comment:
The kernel never use `Unspec`, if that really happens, it should fit into `Other`.

Reply:
I think we discussed this before. In that case, we should remove `Unspec` from the others Info* to keep consistency.